sql/entity_mapping.sql

```diff
@@ -1,17 +1 @@
-INSERT IGNORE INTO entity_mapping
-    (uuid,entity_mapping_type_id,entity1_uuid,entity2_uuid)
-VALUES
-    ('710e53d4-806a-11e9-bc42-526af7764f64',5,'d91e7c8a-0ebe-4792-a332-a175d881fe3a','df204697-a6c6-4768-a4e8-7082940fc8eb'),
-    ('710e58c0-806a-11e9-bc42-526af7764f64',5,'dc0c872a-97d5-4731-aa00-ab8dd712e761','ee0644c1-53e1-4c8c-bcd9-2308c608be60'),
-    ('710e5a32-806a-11e9-bc42-526af7764f64',5,'9eb68046-b654-4884-b560-d1275caa84a7','8568c575-16fc-4330-a3bb-c0460c15a298'),
-    ('710e5b5e-806a-11e9-bc42-526af7764f64',5,'19a762fa-143b-41fe-8ec0-d22630234863','0d414ccb-55a2-480f-93ff-c2af7061f0ba'),
-    ('710e5f1e-806a-11e9-bc42-526af7764f64',5,'1cb46ca4-cf55-4512-9058-eccd042d3a60','4dd3bdc2-38ab-450e-b36f-784ed6f8427d'),
-    ('710e6054-806a-11e9-bc42-526af7764f64',5,'9b5f2493-fa75-44f9-80e4-bb389121ea0d','0d9bf0c2-39f5-489a-a660-927f1ac97b7b'),
-    ('710e6298-806a-11e9-bc42-526af7764f64',5,'a71de1a2-aa43-496a-a533-13f47fad0129','1144b765-2e6c-47b9-bb2d-e059ad1680a8'),
-    ('710e66ee-806a-11e9-bc42-526af7764f64',5,'0cec2833-2914-4516-b6dc-2c8d58d924ec','0607712c-aced-44da-8534-72d639bd9994'),
-    ('710e6838-806a-11e9-bc42-526af7764f64',5,'660e38b5-2599-4ff5-b3fe-5c3fe0a5cd7e','d1757c49-d3eb-4b36-bb2c-1d3faf238b20'),
-    ('710e6964-806a-11e9-bc42-526af7764f64',5,'74c0393a-f5dd-4f4d-98bb-cedda584349b','c6e23237-5a0d-438a-9381-ac403d30054f'),
-    ('710e6d1a-806a-11e9-bc42-526af7764f64',5,'f0a00d58-eeb8-4454-a41c-c8c2664313ec','79157d5d-1e45-407d-bf15-ae8dc1674a70'),
-    ('710e6e6e-806a-11e9-bc42-526af7764f64',5,'976817df-1c8f-46da-8568-f61e5093f814','4dbf4014-4f2c-4c74-ad3e-2f6df3f60c5d'),
-    ('710e6f9a-806a-11e9-bc42-526af7764f64',5,'eda4bd55-7781-4a4d-b02a-40d7488ae586','c7b372e3-ee2d-4e1e-b525-b89372399bbe'),
-    ('710e713e-806a-11e9-bc42-526af7764f64',5,'bc475481-050a-4938-b00a-e8dc6b9b4c01','0efa13f3-4351-4e9c-91fd-1d33a1833028');
+DELETE FROM entity_mapping;
```

Comment:
@prashantsable1991 There are 126 rows in `entity_mapping` in dev, the above INSERT added only 14 new lines; so the delete query will remove records that are possibly used elsewhere. We only need to remove the records for the  uuids added part of COM-37.